src/com/marketplace/service/review/ReviewRepresentation.java

```diff
@@ -1,5 +1,41 @@
 package com.marketplace.service.review;
 
-public class ReviewRepresentation {
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+
 
+@XmlRootElement(name = "Review")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "")
+
+public class ReviewRepresentation {
+	
+	private int reviewID;
```

Comment:
Missing some fields from the `Review` object. Checkout `Review.java` to see what's missing